docs/redoc/src/main/scala/sttp/tapir/redoc/Redoc.scala

```diff
@@ -8,7 +8,13 @@ object Redoc {
 
   val defaultRedocVersion = "2.0.0-rc.56"
 
-  def redocHtml(title: String, specUrl: String, redocVersion: String = defaultRedocVersion): String = s"""
+  def redocHtml(
+    title: String,
+    specUrl: String,
+    redocVersion: String = defaultRedocVersion,
+    redocOptions: String = "",
+    redocThemeOptionsJson: String = "{}"
```

Comment:
sorry for picking on this again, but maybe it would be more Scala-like to represent these as `Option`s? then if the option is defined, we just trust the user know what they're doing and paste in whatever is in there

Reply:
Oh yeah, can do.